llmfoundry/data/finetuning/tasks.py

```diff
@@ -249,11 +250,13 @@ def slice_out_last_turn(
             full_conversation = tokenizer.apply_chat_template(
                 messages_through_current_turn,
                 tokenize=False,
+                date_string=get_date_string(),
```

Comment:
does this not need to be conditional? Or is it just totally ignored if the template doesn't have a date string?

Reply:
it's totally ignored, the kwargs are just used to make variables available for the chat template

it's also not possible to inspect if the `apply_chat_template` function has a `date_string` kwarg because it's just passed as part of kwargs, there's no separate arg name